spec/bulk_processor/csv_processor/row_processor_spec.rb

```diff
@@ -19,7 +19,7 @@ def primary_keys
 
   describe '#result' do
     subject do
-      row = { 'name' => name, 'ssn' => '0' }
+      row = CSV.parse("name,ssn\n#{name},0", headers: true).first
       TestRowProcessor.new(row, row_num: 3, payload: {})
```

Comment:
Looks like it doesn't matter, but it's odd that `run_num: 3` when there's only one data row